Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Utils/FrameCapturePathManagementBus.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Preprocessor/Enum.h>
+#include <AzCore/std/string/string.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        class FrameCapturePathManagementRequests
+            : public EBusTraits
```

Comment:
It's not the only way to expose the functions to Python. Free functions and classes are also supported. But this seems consistent with the rest of the system.

Reply:
It should be possible to just use a class instead of the an ebus. FrameCaptureSystemComponent will have it either way. But I feel it is more consist to call a capture request and then a compare request, instead of calling compare in a different way.